pages/fpki_truststores.md

```diff
@@ -4,43 +4,50 @@ title: Trust Stores
 permalink: /truststores/
 ---
 
-This guide provides a high-level overview of what application trust stores are, a list of commonly used application trust stores, and where to find them.
+This **Trust Stores** guide provides a high-level overview of what **Trust Stores** are, a list of commonly used application trust stores, and where to find them.
 
-## What is an Application Trust Store?
-There are millions of identity (certificates issued to People) and device certificates in the world currently, and the list constantly changes as some certificates are revoked and others are issued - far too many for your computer to maintain an up to date list.  
 
-Instead, applications keeps a list of **root** certificates which it trusts to issue certificates.  When you are presented with an identity or device certificate from a PIV credential, website, email, or some other digital item, your system or application will check to see if the certificate chains to one of the trusted **roots** in the application trust store. The trust store may maintain an indepenet list of trusted root certificates which comply with the individual trust store requirements. 
+## What is a Trust Store?
+There are millions of identity (certificates issued to people) and device certificates issued in the world today, and the list constantly changes as some certificates are revoked and others are issued--far too many for your computer to maintain an up-to-date list.  
 
-This list of trusted root certificates are contained within what is known as a **Trust Store** in either an application or operating system. 
+Instead, each application and the operating system (OS) on your computer keep a list of **trusted root certificates** in a **Trust Store** within the specific application or the OS. From their list of trusted root certificates, the application or the OS issues certificates to people (i.e., identities) and devices. When your computer is presented with an identity or device certificate from a PIV credential, website, e-mail, or other digital item, either an application or the OS will verify whether or not the certificate relates ("chains") to one of the trusted root certificates in the Trust Store. (**Note:** Not only do applications and OSs keep lists of trusted root certificates--Trust Stores themselves sometimes keep a list of trusted root certificates that comply with the Trust Store's requirements.) 
```

Comment:
remove this line - it's confusing: 

> From their list of trusted root certificates, the application or the OS issues certificates to people (i.e., identities) and devices.

Remove the ** Note: ** sentence too.  Confusing.